src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -73,6 +108,8 @@
       <PreviouslySourceBuiltNuGetSourceName>previously-source-built</PreviouslySourceBuiltNuGetSourceName>
       <ReferencePackagesNuGetSourceName>reference-packages</ReferencePackagesNuGetSourceName>
       <SourceBuildSources>$(SourceBuildSources);$(PrebuiltNuGetSourceName);$(PreviouslySourceBuiltNuGetSourceName);$(ReferencePackagesNuGetSourceName)</SourceBuildSources>
+      <BuildCommand>$(BuildCommand) /p:RestoreConfigFile=@(NuGetConfigFiles)</BuildCommand>
```

Comment:
This won't work if NuGetConfigFiles contains more than one item. Should the item be changed to a property?

Reply:
If we can't flow in multiples, then I think the infrastructure to support multiple should be removed to eliminate any confusion if someone tries to use it.

Reply:
I will update this - thanks.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/55d2441f88af313329b757886fc4d72cf7c177c8